src/core_io.h

```diff
@@ -37,11 +37,6 @@ bool DecodeHexBlockHeader(CBlockHeader&, const std::string& hex_header);
  */
 bool ParseHashStr(const std::string& strHex, uint256& result);
 std::vector<unsigned char> ParseHexUV(const UniValue& v, const std::string& strName);
-
```

Comment:
I think this means you can remove the `struct PartiallySignedBitcoinTransaction` forward declare from this file.

Reply:
yep, done